python/setup.py

```diff
@@ -315,11 +315,9 @@ def get_packages(self):
         "dm_tree",
         "gymnasium==1.0.0",
         "lz4",
-        "scikit-image",
+        "ormsgpack==1.7.0",
```

Comment:
Get rid of 3, add 1 :)

Reply:
@sven1977 Do you think we should use `ormsgpack` for our offline RL API? We use right now `msgpack` with `msgpack-numpy`.

Reply:
I'm not 100% sure yet. This is still in experimental stage. `ormsgpack` is faster and natively supports numpy. That's good. But `msgpack-numpy` might produce smaller files. We'll have to see what outweighs what.